Muhammed-Rahif/Notpad
# Pull Request: Offline Font Support for SUSE Font

# Pull Request: Offline Font Support for SUSE Font

## Description

This pull request adds offline font support for the SUSE font to enhance accessibility and performance in our application. With this implementation, users can access the SUSE font without relying on an internet connection, ensuring a consistent user experience.

## Changes Made

- Added offline font files for the SUSE font to the project.
- Updated the CSS to include the offline font source.
- Ensured compatibility across various browsers.

## Why This Change is Necessary

Implementing offline font support allows users to access our application seamlessly without an internet connection. This is particularly beneficial in environments with limited connectivity, enhancing overall user experience.

## How to Test

1. Pull the latest changes from this branch.
2. Open the application in your preferred browser.
3. Verify that the SUSE font is loaded correctly without an internet connection.
4. Check for any styling issues or errors in the console.
